src/index.js

```diff
@@ -40,6 +40,12 @@ const _init = Symbol('init');
 const _defineSetting = Symbol('_defineSetting');
 const _internalSet = Symbol('_internalSet');
 
+const instances = [];
+
+process.on('exit', () => {
+  instances.forEach(instance => instance.close());
+});
```

Comment:
A `for ... of` loop would be nice here. ~~`forEach` is ugly~~

Reply:
Ffs, this doesn't matter at all...

Reply:
I agree with @RealShadowNova haha